keps/sig-api-machinery/20180415-crds-to-ga.md

```diff
@@ -157,15 +162,37 @@ TODO: complete this list
 
 ### e2e tests for GA
 
-* Custom Resources should be readable and writable at all available versions (test for round trip-ability)
 * A Custom Resource watch should terminate if its CustomResourceDefinition is deleted or updated
 
 ### Scale Targets for GA
 
-* TODO quantify: Read/write latency of CRDs within X% of native Kubernetes types
-* TODO quantify: Latency degrades less than X% for up to 100k Custom Resources per CRD kind
-* TODO quantify: Webhook conversion QPS of a noop converter is within X% of QPS with no webhook
-* Coordinate with sig-scalability
+Given that (a) the performance and scalability of conversion webhooks are the
+responsibly of their author, and (b) custom resources can be arbitrarily large,
+Kubernetes cannot offer an SLOs for custom resources like we do for
+Kubernetes native types (see [API call latency
+SLIs/SLOs](https://github.com/kubernetes/community/blob/master/sig-scalability/slos/api_call_latency.md#api-call-latency-slisslos-details).
+
+Instead, we will set a scale target for custom resources using the same approach as the other [kubernetes
+thresholds](https://github.com/kubernetes/community/blob/master/sig-scalability/configs-and-limits/thresholds.md#kubernetes-thresholds):
+
+- namespace scope: 2000
+- cluster scope: 10000
+
+such that apply when these conditions are met:
```

Comment:
rephrase? not sure what this is saying

Reply:
Re-organized this so it might read more simply.